molecule/unofficial/molecule.yml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
The run list for `test` by default includes:

```

└── unofficial
    ├── destroy
    ├── dependency
    ├── syntax
    ├── create
    ├── prepare
    ├── converge
    ├── idempotence
    ├── lint
    ├── side_effect
    ├── verify
    └── destroy
```

I would remove `idempotence`, `prepare`, and  `side_effect`

Reply:
Good call on removing the unnecessary steps, will do that!